app/web/src/features/dashboard/CommunityBrowser.stories.tsx

```diff
@@ -0,0 +1,45 @@
+import { Meta, Story } from "@storybook/react";
```

Comment:
This is broken when I checked :(

![Screenshot 2021-09-19 at 01 21 40](https://user-images.githubusercontent.com/13669362/133911666-965b0f00-b2f2-49b1-8fdf-64a5e7223961.png)

Missing mock for `mockedService.communities.getCommunity`? Intriguing how this worked before without it 🤔